.github/actions/setup/action.yml

```diff
@@ -48,7 +68,7 @@ runs:
         version: ${{ inputs.solana }}
         cache: true
     
-    - name: Cache cargo dependencies
+    - name: Cache Cargo Dependencies
       if: ${{ inputs.cargo-cache-key }}
       uses: actions/cache@v4
       with:
```

Comment:
Oh for some reason I assumed this cache extraction was always there but it's been changed in a previous PR. I'm looking at it in more details right now and this actually break the caching mechanism by saving in both the regular cache and the test cache regardless on the job we're in. I'm gonna try and keep this extracted but there was a reason why I didn't do this to begin with as almost every job caches these in a slightly different way.